### PR TITLE
[Gradle] Set working directory before fetching for gradle wrapper script

### PR DESCRIPTION
**Pull request by benabbon**

### CODE REVIEW

**Comment by bryanmacfarlane** [comment]
@jeffyoung - can you review?

File: Tasks/Gradle/gradletask.ts [hunk 1]
```diff
@@ -151,16 +151,19 @@ async function run() {
...
+		
         // Configure wrapperScript
         let wrapperScript: string = tl.getPathInput('wrapperScript', true, true);
         wrapperScript = configureWrapperScript(wrapperScript);
 
-        // Set working directory
-        let workingDirectory: string = tl.getPathInput('cwd', false, true);
         if (!workingDirectory) {
             workingDirectory = path.dirname(wrapperScript);
         }
```
Comment (DavidStaheli) [inline Tasks/Gradle/gradletask.ts]: Here in the `if (!workingDirectory)` block, we still need `tl.cd(workingDirectory)` after `workingDirectory = path.dirname(wrapperScript);`

Comment (benabbon) [inline Tasks/Gradle/gradletask.ts]: just added it

**Comment by DavidStaheli** [comment]
Thanks very much for the contribution, @benabbon!

**Pull request merged**